tests/rptest/tests/timequery_test.py

```diff
@@ -323,10 +323,20 @@ def _do_test_timequery(self, cloud_storage: bool, batch_cache: bool,
                              batch_cache=batch_cache)
         if spillover:
             # Check that we are actually using the spillover manifest
-            bucket = BucketView(self.redpanda)
-            manifest = bucket.manifest_for_ntp("tqtopic", 0)
-            assert manifest["archive_start_offset"] == 0
-            assert manifest["start_offset"] > 0
+            def check():
+                try:
+                    bucket = BucketView(self.redpanda)
+                    manifest = bucket.manifest_for_ntp("tqtopic", 0)
+                    return manifest.get(
+                        "archive_start_offset",
+                        -1) == 0 and manifest.get("start_offset", -1) > 0
```

Comment:
nit: could use `bucket.get_spillover_metadata` and check the result is not empty.

Reply:
done